src/Jarvis.Addin.Files/FileProvider.cs

```diff
@@ -55,7 +59,33 @@ protected override Task ExecuteAsync(FileResult result)
 
                 if (result.Path.Scheme == "shell")
                 {
-                    Process.Start(path);
+                    var descriptionPath = result.Description.Replace("/", "\\");
```

Comment:
Using the `FileResult.Description` only works in some cases. Where a `lnk`-file has been indexed, this will be the shortcut's description.

Reply:
Yes, that's what I feared.